src/Settings/SettingsValidator.php

```diff
@@ -103,6 +105,10 @@ private function has_valid_email() {
 	 * @return bool
 	 */
 	private function has_valid_phone(): bool {
+		// Not required.
+		if( empty( $this->settings->get_phone_number() ) ) {
+			return true;
+		}
```

Comment:
@bplv112 Same comments from email logic above applies here.

Can you please add a space after `if` here as well?